### ISSUE TITLE
Alerts stop after some time of use

### ISSUE THREAD

**Issue by cw-pratik** [Graylog2/graylog2-server#3984]
I have tried to setup alerts on streams in graylog. It was set to trigger when there were more than 10 messages in the stream in the last one minute. Initially for a month or two it worked perfectly. After that the alerts stopped. 

Then I tried a lot of things like restarting graylog, restarting the server itself but nothing worked.

Then I changed the trigger condition to check for the last 2 minutes instead of one. And the triggers started to work again. 

* Graylog Version: 2.2.2
* Elasticsearch Version: 2.4.1
* MongoDB Version: 3.0.14
* Operating System: Ubuntu 14.04
* Browser version: chrome 59

**Comment by edmundoa** [comment]
Hi @cw-pratik,

With the provided information it is hard to reproduce the issue. Did you see any errors on your Graylog server logs when the alert was not working? Were you using Graylog 2.2.2 the whole time? As you have more information about your system, would you be so kind as to also check if the issue you described may be related to https://github.com/Graylog2/graylog2-server/issues/3884?

Thank you!

**Comment by cw-pratik** [comment]
Hi @edmundoa,

I checked, The issue is not related to #3884. 
I might have upgraded the server once. (From 2.2 or 2.2.1).
Unfortunately I don't have the logs as we realized after a long time that the alerts are not coming and till then the logs were flushed.
The thing is that happened for the second time. We had the same issue on an older version of graylog. At that time also we had changed the trigger condition to check for the last two minutes instead of one and alerts started to work. We then were in the process of switching the AWS region so we did a fresh install of 2.2 version and things started to work for sometime. 

Can this have something to do with mongodb as all the metadata is stored there?
Is there any thing else that I can do to check why It is not working. As on my system alerts are still not being triggered when the trigger condition checks for last one minute. 

Thanks.
